src/c++/library/grpc_client.cc

```diff
@@ -623,7 +623,7 @@ Error
 InferenceServerGrpcClient::LoadModel(
     const std::string& model_name, const Headers& headers,
     const std::string& config,
-    const std::map<std::string, std::string>& encoded_files)
+    const std::map<std::string, std::vector<char>>& files)
```

Comment:
`const std::vector<char>>& files`

Reply:
Map is needed because the key is used to specify the file path to store the file content, which constructs the override model directory.

Reply:
Oh I misread this, didn't realize the vector was the 2nd part of map type.